pkg/controllers/clusterresourceplacement/controller_test.go

```diff
@@ -2017,6 +2079,57 @@ func TestGetOrCreateClusterResourceSnapshot_failure(t *testing.T) {
 	}
 }
 
+func TestSplitSelectedResources(t *testing.T) {
+	serviceResourceContent := *serviceResourceContentForTest(t)
+	deploymentResourceContent := *deploymentResourceContentForTest(t)
+	secretResourceContent := *secretResourceContentForTest(t)
+	tests := []struct {
+		name                       string
+		selectedResourcesSizeLimit int
+		selectedResources          []fleetv1beta1.ResourceContent
+		wantSplitSelectedResources [][]fleetv1beta1.ResourceContent
+	}{
+		{
```

Comment:
please add a test case when a resource content size is greater than the limit already.